### ISSUE TITLE
Add note for mod authors on how to "flag" intentional ITMs in "5. Mod Cleaning and Error Checking"

### ISSUE THREAD

**Issue by Freso** [TES5Edit/docs#1]
@eFrysTon has mentioned a couple of times on the xEdit Discord mod authors should make a change to a record's Editor ID/`EDID` to prevent an intentional ITM from being flagged as ITM in xEdit when cleaning. It would be good to include this in the documentation so mod authors can be pointed to it (or simply come across when reading the xEdit documentation).

Either https://tes5edit.github.io/docs/5-mod-cleaning-and-error-checking.html#s_5-3-1 or https://tes5edit.github.io/docs/5-mod-cleaning-and-error-checking.html#Apendix-A seem like good places to point to, but it might also be better with a new section (appendix) with "tips for mod authors" or similar. Either way, it should preferably have an anchor so the section can be linked to directly.

**Comment by Sharlikran** [comment]
I like your tips for mod authors idea as it isn't directly related to what is an ITM or how do I remove an ITM. Rather it is a way for mod authors to prevent intentional ITMs from being removed. I appreciate the suggestion and I'll find a place for this and I will add an anchor as well. 

Thank you.

**Comment by ElminsterAU** [comment]
Just want to point out that this is primarily aimed at TES5/SSE and FO4, as EditorIDs are relatively meaningless in these beyond being a convenient label for the CK. Also, for these games, I've defined EditorIDs as case sensitive, but with an "cpOverride" conflict priority. That means a simple change in case is enough to show up as a change (preventing it from being detected as an ITM) but even if multiple different mods end up changing the EditorID of the same record, it will never be showing up as more than "override without conflict".

I have no made any changes to the record definitions for older games yet (that will require more contemplate as EditorIDs are directly used in scripts in these).

**Comment by Sharlikran** [comment]
GMST is also driven by edid isn't it?

**Comment by ElminsterAU** [comment]
Yes, but the engine is case insensitive in regards to EditorIDs, so just changing the case should make no difference to the game.
